# DP: fix testcases that triggered -Wunused-result with glibc
# DP: Author: Steve Beattie <steve.beattie@canonical.com>
---
 src/gcc/testsuite/c-c++-common/cilk-plus/AN/comma_exp.c |    2 +-
 src/gcc/testsuite/c-c++-common/tsan/fd_pipe_race.c      |    1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

Index: b/src/gcc/testsuite/c-c++-common/tsan/fd_pipe_race.c
===================================================================
--- a/src/gcc/testsuite/c-c++-common/tsan/fd_pipe_race.c
+++ b/src/gcc/testsuite/c-c++-common/tsan/fd_pipe_race.c
@@ -1,5 +1,5 @@
 /* { dg-shouldfail "tsan" } */
-/* { dg-additional-options "-ldl" } */
+/* { dg-additional-options "-Wno-unused-result -ldl" } */
 
 #include <pthread.h>
 #include <unistd.h>
Index: b/src/gcc/testsuite/c-c++-common/cilk-plus/AN/comma_exp.c
===================================================================
--- a/src/gcc/testsuite/c-c++-common/cilk-plus/AN/comma_exp.c
+++ b/src/gcc/testsuite/c-c++-common/cilk-plus/AN/comma_exp.c
@@ -1,5 +1,5 @@
 /* { dg-do run } */
-/* { dg-options "-fcilkplus" } */
+/* { dg-options "-fcilkplus -Wno-unused-result" } */
 
 #include <stdlib.h>
 
